### DISCUSSION TITLE
Inquirer vs Enquirer

### DISCUSSION THREAD

**Discussion by brandonbechtel** [nrwl/nx#13902] [General]
Hi, I just wanted to chime in on this and get some thoughts. I know Nx migrated over to `enquirer` a little over a year ago due to `inquirer`'s dependency on RxJS and the size/performance impact. Just looking at the two repos today, it looks like `inquirer` is smaller (images below, though I don't know if the unpacked size metric on the npm repo page includes dependencies), but it is also being more actively developed. Is the intent of Nx to stick with `enquirer`?

For reference, this came up as something that could impact issue #9038 

<img width="352" alt="image" src="https://user-images.githubusercontent.com/115498161/208119955-ccecc9f5-8486-4462-a95d-b1b3e1884099.png">
<img width="344" alt="image" src="https://user-images.githubusercontent.com/115498161/208119986-005b2970-f454-4d98-9705-887d274b8eea.png">

**Comment by jskrzypek** [comment]
@brandonbechtel FWIW, I think the code frequency is a good thing to consider in this discussion as well. Inquirer seems to be more actively maintained:


##### [Enquirer Code Frequency](https://github.com/enquirer/enquirer/graphs/code-frequency)
<a href="https://github.com/enquirer/enquirer/graphs/code-frequency"><img width="1202" alt="Enquirer Code Frequency" src="https://user-images.githubusercontent.com/1513265/208516940-bee0208c-bc44-4261-8140-bdcaf57bef67.png"></a>

##### [Inquirer.js Code Frequency](https://github.com/SBoudrias/Inquirer.js/graphs/code-frequency)
<a href="https://github.com/SBoudrias/Inquirer.js/graphs/code-frequency"><img width="1202" alt="Inquirer.js Code Frequency" src="https://user-images.githubusercontent.com/1513265/208517098-731520c0-1b8b-46fa-8adc-49b6a6f2be50.png"></a>

**Comment by jskrzypek** [comment]
Another aspect to consider is extensibility, and whether plugins are needed for comparable functionality, e.g. Enquirer includes autocomplete in it's builtin prompts, whereas you get autocomplete for Inquirer.js via a plugin: https://github.com/mokkabonna/inquirer-autocomplete-prompt
